<ui version="4.0" >
 <class>SummaryWordWidget</class>
 <widget class="QWidget" name="SummaryWordWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>236</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="languageLabel" >
     <property name="text" >
      <string>Language</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>wordEntry</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLineEdit" name="wordEntry" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Lesson</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Word type</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>wordTypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Pronunciation</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>pronunciationEntry</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLineEdit" name="pronunciationEntry" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Example</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>exampleEntry</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QLineEdit" name="exampleEntry" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Paraphrase</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>paraphraseEntry</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QLineEdit" name="paraphraseEntry" />
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Comment</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>commentEntry</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QLineEdit" name="commentEntry" />
   </item>
   <item row="2" column="2" >
    <widget class="KComboBox" name="wordTypeComboBox" />
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="lessonLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
